app/src/main/res/values/strings.xml

```diff
@@ -1,81 +1,179 @@
-    <!-- Parameter will be replaced with the app name -->
+    <!-- LOCALIZATION NOTE (permission_microphone): '%1$s' will be replaced at runtime with the app name.
+         The string 'permission_microphone' will be displayed in a dialog box presented to the user
+         when the application needs to request permission to access the device's microphone.-->
     <string name="permission_microphone">Will you allow %1$s to use your microphone?</string>
-    <!-- Parameter will be replaced with the app name -->
+    <!-- LOCALIZATION NOTE (permission_camera_and_microphone): '%1$s' will be replaced at runtime with the app name.
+         The string 'permission_camera_and_microphone' will be displayed in a dialog box presented to the user
+         when the application needs to request permission to access the device's camera and microphone. -->
     <string name="permission_camera_and_microphone">Will you allow %1$s to use your camera and microphone?</string>
-    <!-- Parameter will be replaced with the app name -->
+    <!-- LOCALIZATION NOTE (permission_location): '%1$s' will be replaced at runtime with the app name.
+         The string 'permission_location' will be displayed in a dialog box presented to the user
+         when the application needs to request permission to access the device's geographic location. -->
     <string name="permission_location">Will you allow %1$s to access your location?</string>
+    <!-- LOCALIZATION NOTE (permission_notification): '%1$s' will be replaced at runtime with the app name.
+         The string 'permission_notification' will be displayed in a dialog box presented to the user
+         when the application needs to request permission to send system notifications. -->
     <!-- Parameter will be replaced with the app name -->
     <string name="permission_notification">Will you allow %1$s to send notifications?</string>
-    <!-- Parameter will be replaced with the app name -->
+    <!-- LOCALIZATION NOTE (permission_read_external_storage): '%1$s' will be replaced at runtime with the app name.
+         The string 'permission_read_external_storage' will be displayed in a dialog box presented to the user
+         when the application needs to request permission to access the device's external storage such as the
+         sdcard. -->
     <string name="permission_read_external_storage">Will you allow %1$s to read your external storage?</string>
+    <!-- LOCALIZATION NOTE (): FIXME It doesn't look like this string is actually used -->
     <string name="speak_now">Speak now</string>
+    <!-- LOCALIZATION NOTE (settings_version): '%1$s' will be replaced at runtime with the application
+         version number. The string 'settings_version' is displayed in the settings dialog box and how
+         the user can know what version of the application they are currently running. -->
     <string name="settings_version">version %1$s</string>
```

Comment:
`version` should probably be uppercased, right?

Reply:
it's technically correct, but perhaps we could say `the version of Firefox Reality` instead of `the application version number`?

Reply:
I was trying not to include `Firefox Reality` any where to make a product name change easier and to prevent confusion in the future. For example Firefox Focus is called Klar in Germany.